iothub/device/src/ModuleClient.cs

```diff
@@ -433,19 +437,24 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
 
         /// <summary>
         /// Set a callback that will be called whenever the client receives a state update
-        /// (desired or reported) from the service.  This has the side-effect of subscribing
```

Comment:
Why did we chose to remove this explanation?

Reply:
We did not remove. It is part of remarks